rviz_common/src/rviz_common/visualization_manager.cpp

```diff
@@ -435,8 +437,7 @@ void VisualizationManager::onUpdate()
   }
 
   frame_count_++;
-
-  if (render_requested_ || wall_dt > 0.01) {
+  if (render_requested_ || wall_diff > std::chrono::milliseconds(10)) {
```

Comment:
@wjwwood now _that_ looks a lot more sensible to me, at least.

Sorry again, that was pretty silly.

Reply:
No worries, I've made sillier mistakes :)